src/@utils/schaclSchema.ts

```diff
@@ -0,0 +1,55 @@
+import { retrieveShaclSchema } from '@utils/aquarius'
+import { ShaclSchemaField } from '@context/MarketMetadata/_shaclType'
+
+import { capitalizeFirstLetter } from '@utils/textTransform'
+
+function getMinMax(valueField: string, field: ShaclSchemaField) {
+  let min = field.minLength
+  const max = field.maxLength
+
+  // TODO: remove once all fields in schema has minLength property
```

Comment:
I don't see an issue in aquarius for doing this? If it's not done yet we should add an issue. It would also be good to note the issue in the comment so we can track when it is done

Reply:
yeah, this too is old comment. It has been removed, as this was already fixed on aqua.